go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/hashicorp/terraform-plugin-go
 
-go 1.24
+go 1.23.0
```

Comment:
Looks like the Go module was inadvertently bumped by `go-plugin@1.7.0`, I'm reverting that change for the moment, so we can come back and bump the version along with the CI -> https://github.com/hashicorp/terraform-providers-devex-internal/issues/189